src/scripts/monitor.ts

```diff
@@ -0,0 +1,36 @@
+import { config as dotEnvConfig } from "dotenv";
+import { ethers } from "ethers";
+
+import * as ABI from "../artifacts/contracts/KeyperPayer.sol/KeyperPayer.json";
+
+dotEnvConfig({ override: true });
+
+const ContractAddress = process.env.CONTRACT_ADDRESS || "";
```

Comment:
nitpick: You can shorten the whole thing
```
const { CONTRACT_ADDRESS = "", KEYPER_ADDRESS = "", RPC_URL = "" } = process.env;
```
and then validate with:
```
function validate(name, value) {
  if (!value) throw `${name} env variable should be set`;
  if (!ethers.isAddress(value)) throw `${name} env variable is not an address`;
}
```
validate("CONTRACT_ADDRESS", CONTRACT_ADDRESS);

But fine for me.